Sergey/Medium/index.html

```diff
@@ -0,0 +1,188 @@
+                            <h4>Maya Kosoff</h4>
+                        </div>
+                        <h2>A Tale Of Two Kinds of Media Workers</h2>
+                        <div class="date">
+                            <span>Oct 20</span>
+                            <span> | </span>
+                            <span>3 min read</span>
+                        </div>
+                    </div>
+                </div>
+                <div class="items_number">
+                    <div class="number">
+                        <span>04</span>
+                    </div>
+                    <div class="item">
+                        <div class="fuck">
+                            <i class="far fa-circle"></i>
+                            <h4>Robert Roy Britt in Elemental</h4>
+                        </div>
+                        <h2>10 Signs the Pandemic Is About to Get Much Worse</h2>
+                        <div class="date">
+                            <span>Oct 19</span>
+                            <span> | </span>
+                            <span>8 min read</span>
+                        </div>
+                    </div>
+                </div>
+                <div class="items_number">
+                    <div class="number">
+                        <span>05</span>
+                    </div>
+                    <div class="item">
+                        <div class="fuck">
+                            <i class="far fa-circle"></i>
+                            <h4>Tristan Joshua Alba in Towards Date Science</h4>
+                        </div>
+                        <h2>5 Reasons why I'm learning Web Development, as a Date Scientist</h2>
+                        <div class="date">
+                            <span>Oct 20</span>
+                            <span> | </span>
+                            <span>8 min read</span>
+                        </div>
+                    </div>
+                </div>
+                <div class="items_number">
+                    <div class="number">
+                        <span>06</span>
+                    </div>
+                    <div class="item">
+                        <div class="fuck">
+                            <i class="far fa-circle"></i>
+                            <h4>Kat Rosenfield  in Arc Digital</h4>
+                        </div>
+                        <h2>White Woman Need Not Apply</h2>
+                        <div class="date">
+                            <span>Oct 19</span>
+                            <span> | </span>
+                            <span>5 min read</span>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="tegs">
+        <div class="tegs_conteiner">
+            <div class="tegs_container1">
+                <p>1</p>
+            </div>
+            <div class="tegs_container2">
+                <div class="tegs_container2_top">
```

Comment:
Very often the word "container" is repeated. Try to describe what is happening in the container with the name, less numbers.

Reply:
> Very often the word "container" is repeated. Try to describe what is happening in the container with the name, less numbers.
Nothing else came to mide(